src/java.base/share/classes/java/lang/invoke/NativeMethodHandle.java

```diff
@@ -23,34 +23,48 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2024, 2024 All Rights Reserved
```

Comment:
The first year was correct: you started this change in 2023; same for `JavaLangInvokeAccess`.

Reply:
Fixed.